egs/grid/VSR/install.sh

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+stage=1
+
+ICEFALL_DIR="$PWD/../../../"
+AVHUBERT_DIR="$PWD/av_hubert"
+
+. shared/parse_options.sh || exit 1
```

Shellcheck (0.11.0)

[info] Not following: shared/parse_options.sh was not specified as input (see shellcheck -x).

(SC1091)